pkg/volume/gce_pd/attacher.go

```diff
@@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
 	options := []string{}
 	if readOnly {
 		options = append(options, "ro")
+	} else {
+		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
```

Comment:
@thockin @ciwang @saad-ali 
Can we invoke [file -s](http://linux.die.net/man/1/file) on the block device so that we can determine the filesystem type in a portable manner? 
Check out: http://www.thegeekstuff.com/2011/04/identify-file-system-type/
This might be a good PR for ciwang's stack of intern items. Or does "file" not exist in our image?